dask_cuda/utils.py

```diff
@@ -10,7 +32,11 @@
 from typing import Optional
 
 import numpy as np
-import pynvml
+from dask_cuda import DASK_USE_ROCM
```

Comment:
@younseojava  IMO You could simplify

```py
if DASK_USE_ROCM:
    from pyrsmi import rocml as pynvml
else:
    import pynvml
```
to

```python
from pyrsmi import rocml as pynvml
```

as users with CUDA device will likely use the `rapidsai/dask_cuda` package and the NVIDIA `rapidsai` org will likely not accept a version that supports both backends.